composer.json

```diff
@@ -20,7 +20,8 @@
         "silverstripe/cms": "^5",
         "guzzlehttp/guzzle": "^7.5",
         "fzaninotto/faker": "^1.9.2",
-        "silverstripe/vendor-plugin": "^2"
+        "silverstripe/vendor-plugin": "^2",
+        "silverstripe/spamprotection": "^4"
```

Comment:
Let's not just arbitrarily add that module - use `if (class_exists)` to only declare the class if it's needed.

Reply:
Updated